File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.java
class Solution {
    public int maxCoins(int[] piles) {
        int l=0;
        int r=piles.length;
        int c=0;
        Arrays.sort(piles);
        for(int i=0;i<piles.length/3;i++){
            c+=piles[r-2];
            r-=2;
        }return c;
    }
}

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        l,r=0,len(piles)
        c=0
        piles.sort()
        for i in range(r//3):
            c+=piles[r-2]
            l+=1
            r-=2
        return c